Add analyzer comments for Elixir lasagna (#2014)
exercism/website-copy: analyzer-comments/elixir/lasagna/function_reuse.md

# function reuse

> Proper function reuse is crucial to writing maintainable code.

Reviewer: ```suggestion
Reusing a written function can help when writing maintainable code.
```

> Reusing a written function can help when writing maintainable code.

Author: ```suggestion
Reusing an already written function can help when writing maintainable code.
```

Reuse existing functions:

> - `remaining_minutes_in_oven/1` should call `expected_minutes_in_oven/0`

Reviewer: ```suggestion
- `remaining_minutes_in_oven/1` can reuse `expected_minutes_in_oven/0`
```

> - `total_time_in_minutes/2` should call `preparation_time_in_minutes/1`

Reviewer: ```suggestion
- `total_time_in_minutes/2` can reuse `preparation_time_in_minutes/1`
```
